### ISSUE TITLE
Allow gaps in timeline

### ISSUE THREAD

**Issue by ghost** [flrs/visavail#2]
First off: Nice work! This is a very nice library.

I've got a request for a new feature though:
Currently the timeline can have four states:
1. Not started yet
2. Data available
3. No data available
4. Ended

It would be great if you could have intermediate gaps, i.e ranges where you don't know what happened.

This could be useful for cases where you are recording data every other month and within the month you'd want to be able to know if there's data or not and then the next month would be blank (white).

So basically a differentiation between "there's supposed to be data, but no data is available" and "there is no data, but there was never supposed to be any in the first place":

Something like this basically:

![Example](http://i.imgur.com/gGISAsa.png)

If this is already possible, then I didn't find out how to do it.

**Comment by flrs** [comment]
Hi @WACMemphis!

Thank you for your nice and well explained feedback!

I agree, the ability to display time gaps would be useful. I just pushed a commit that serves that purpose. Check out the updated [readme](https://github.com/flrs/visavail/blob/master/README.md) and the [time gap example](https://github.com/flrs/visavail/blob/master/examples/example_timegaps.htm).

Let me know if this is what you were looking for. Thanks!

**Comment by ghost** [comment]
Excellent work! And quick as well. Thanks.
